developer_docs/README.md

```diff
@@ -117,7 +117,12 @@ A complete guide to unit testing is beyond the scope of the p5.js documentation,
 
    If you're continuously changing files in the library, you may want to run `npm run dev` to automatically rebuild the library for you whenever any of its source files change without you having to first type the command manually.
 
-6. Make some changes to the codebase and [commit](https://help.github.com/articles/github-glossary/#commit) them with Git.
+6. Make some changes locally to the codebase and [commit](https://help.github.com/articles/github-glossary/#commit) them with Git.
+
+   ```
+   $ git add -A
```

Comment:
@abhinavsagar I think I'd rather recommend people add only add already tracked files as a default instead of everything in the folder. Does that seem okay to you?

```suggestion
   $ git add -u
```